python/cross_service/aurora_rest_lending_library/library_demo.py

```diff
@@ -289,7 +290,15 @@ def main():
 
     if args.action == "deploy_database":
         print("Deploying the serverless database and supporting resources.")
-        do_deploy_database(cluster_name, secret_name)
+        cluster, secret = create_resources(
```

Comment:
This doesn't look like a formatting change.

Reply:
This is because the file was broken to begin with. Flake8 picked up on a non-existent function as a violation, and when I looked into it the whole thing was incorrect and calling a non-existent function. So, I fixed it.